cmd/der2ascii/main.go

```diff
@@ -49,6 +51,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if *isPem {
+		pemBlock, _ := pem.Decode(inBytes)
+		inBytes = pemBlock.Bytes
```

Comment:
If `pemBlock` is nil, probably this ought to print a nice error rather than panic. :-)